pkg/kt/snmp_test.go

```diff
@@ -20,7 +20,8 @@ func TestIsPollReady(t *testing.T) {
 	assert.False(t, mib.IsPollReady()) // Skip the 2nd.
 	assert.False(t, mib.IsPollReady()) // Skip the 2nd.
 
-	mib.lastPoll = time.Now().Add(-1 * 10 * time.Second) // reset.
-	assert.True(t, mib.IsPollReady())
-	assert.False(t, mib.IsPollReady()) // And now its false.
+	// These are failing randomly in CI, commenting out for now.
```

Comment:
Should we just remove the flaky test or do you want to keep it here to try to resolve it later on?